.config/forest.dic

```diff
@@ -11,7 +11,7 @@ Filecoin
 IPLD
 JSON
 JWT
-Merkle
+merkle
```

Comment:
In most cases I think `Merkle` is correct (capitalized, it's someone's name I think).

Reply:
That makes, fixed up